kratos/solving_strategies/strategies/line_search_strategy.h

```diff
@@ -216,6 +255,14 @@ class LineSearchStrategy
     ///@name Protected member Variables
     ///@{
 
+    Parameters mSettings;
```

Comment:
Can you add comments describing the variables?

Reply:
done